src/java/boa/compiler/SymbolTable.java

```diff
@@ -273,7 +278,7 @@
 
 		// self casts
 		globalFunctions.addFunction("bool", new BoaFunction(new BoaBool(), new BoaType[] { new BoaBool() }, "${0}"));
-		globalFunctions.addFunction("int", new BoaFunction(new BoaInt(), new BoaType[] { new BoaInt() }, "${0}"));
+		globalFunctions.addFunction("int", new BoaFunction(new BoaInt(), new BoaType[] { new BoaInt() }, "(long)${0}"));
```

Comment:
What is the point of this cast?  This function is a self cast - casting an 'int' to an 'int' in Boa.  This means the argument is already a long and the cast is redundant.